elections/steering/2024/README.md

```diff
@@ -0,0 +1,313 @@
+
+### Decision
+
+- First, the results are privately announced to the incumbent Steering Committee
+members (who are not up for election) and all the candidates.
+
+- The newly elected body will be publicly announced in the scheduled monthly
+[public Steering Committee Meeting].
+
+- Following the meeting, the raw voting results and winners will be published on the
+[Kubernetes Blog].
+
+For more information, definitions, and/or detailed election process, please refer to
+the [Steering Committee Election Charter]
+
+## Nominees
+
+Nominees may be found in the [election app].
+
+[Kubernetes Steering Committee]: https://github.com/kubernetes/steering
+[Steering Committee Charter]: https://github.com/kubernetes/steering/blob/master/charter.md
+[current steering committee backlog]: https://github.com/orgs/kubernetes/projects/40
+[governance meeting video]: https://www.youtube.com/watch?v=ltRKXLl0RaE&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ&index=23
+
+[Steering Committee Election Charter]: https://git.k8s.io/steering/elections.md
+[Eligibility for voting]: https://github.com/kubernetes/steering/blob/master/elections.md#eligibility-for-voting
+[Eligibility for candidacy]: https://github.com/kubernetes/steering/blob/master/elections.md#eligibility-for-candidacy
+[limiting corporate campaigning]: https://github.com/kubernetes/steering/blob/master/elections.md#limiting-corporate-campaigning
+[pledge to recuse]: https://github.com/kubernetes/steering/blob/master/elections.md#steering-committee-and-election-officer-recusal
+
+[Condorcet]: https://en.wikipedia.org/wiki/Condorcet_method
+[prior candidate bios]: https://github.com/kubernetes/community/tree/master/elections/steering/2022
+[election officers]: https://github.com/kubernetes/community/tree/master/elections#recommending-election-officers
+[Kubernetes Community Meeting]: https://github.com/kubernetes/community/blob/master/events/community-meeting.md
+[Kubernetes Blog]: https://kubernetes.io/blog/
+
+[devstats-sql]: https://github.com/cncf/devstats/blob/master/metrics/shared/project_developer_stats.sql
+[devstats-dashboard]: https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All
+[Org Members]: https://github.com/kubernetes/community/blob/master/community-membership.md
+[Elekto]: https://elekto.dev
+[election app]: https://elections.k8s.io
+[Elekto voting documentation]: https://elekto.dev/docs/voting/
+[voters.yaml]: https://github.com/kubernetes/community/blob/master/elections/steering/2024/voters.yaml
+[election page]: https://elections.k8s.io/app/elections/steering---2023
+[voter exception form]: https://elections.k8s.io/app/elections/steering---2023/exception
+[public Steering Committee Meeting]: https://github.com/kubernetes/steering/#meetings
+[Eligible voters]: https://github.com/kubernetes/community/tree/master/elections/steering/2023#eligibility
```

Comment:
Some link needs update to 2024.

Reply:
Not doing any of this right now, that's up to the Officers once selected.